dotnet-runtime-90/snap/snapcraft.yaml

```diff
@@ -0,0 +1,84 @@
+name: dotnet-runtime-90
+base: core22
+adopt-info: dotnet-runtime
+summary: .NET 9.0 Runtime.
```

Comment:
**nitpick:** the other runtime `snapcraft.yaml` files say ".NET Core X.0 Runtime."
note: the "Core is missing"

**suggestion:** 
1. Add the "Core"
    ```suggestion
    summary: .NET Core 9.0 Runtime.
    ```
2. or remove the Core from the other runtime `snapcraft.yaml` files

Reply:
Yes, I have created an issue to track this (#18) as the work is a bit more involved. I want to drop the "Core" from the other `snapcraft.yaml` files, as the runtime is no longer called ".NET Core", and create a more helpful and descriptive `description` entry for all content snaps.

In the meantime, I dropped the "Core" from the .NET 9 Runtime content snap so that the wrong naming isn't "perpetuated" and pushed to the future.

Reply:
Makes sense, resolving this then.